index.js

```diff
@@ -49,16 +49,17 @@ class ReportPortalHelper extends Helper {
 
     return rpClient.startLaunch({
       name: this.config.launchName || suiteTitle,
-      start_time: rpClient.helpers.now(),
       description: this.config.launchDescription || '',
+      attributes: this.config.launchAttributes || [],
+      rerun: this.config.rerun || false,
+      rerunOf: this.config.rerunOf || undefined,
```

Comment:
could you give the references of those?

Reply:
Yes, of course.
I've already added the link [here](https://github.com/PeterNgTr/codeceptjs-rphelper/pull/10/files#diff-04c6e90faac2675aa89e2176d2eec7d8R49).
https://github.com/reportportal/documentation/blob/master/src/md/src/DevGuides/rerun.md#specified-launch